topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -42,19 +43,28 @@ IntelliJ Platform Gradle Plugin 2.x requires the following minimal versions:
 >
 {style="note"}
 
-To apply the IntelliJ Platform Gradle Plugin to a project, add the following entry to the `plugins` block in the <path>build.gradle.kts</path> file:
-
-```kotlin
-plugins {
-  id("org.jetbrains.intellij.platform") version "%intellij-platform-gradle-plugin-version%"
-}
-```
-
+To apply the IntelliJ Platform Gradle Plugin to a project, you need to add the following:
+
+1. Add the `gradlePluginPortal` repository to the <path>settings.gradle.kts</path> file:
+   <code-block lang="kotlin" >pluginManagement {
+   repositories {
+   gradlePluginPortal()
+   }
+   }
+   </code-block>
+2. Add the `plugins` block in the <path>build.gradle.kts</path> file:
+   <code-block lang="kotlin" >plugins {
+   id("org.jetbrains.intellij.platform") version "%intellij-platform-gradle-plugin-version%"
+   }
+   </code-block>
+
+<tip>
 If migrating from the Gradle IntelliJ Plugin 1.x, replace the old `org.jetbrains.intellij` identifier to `org.jetbrains.intellij.platform` and apply its latest `%intellij-platform-gradle-plugin-version%` version.
+</tip>
 
 ### Snapshot Release
 
-To use the latest snapshot versions, add the following to the <path>settings.gradle.kts</path> file:
+To use the latest snapshot versions of this plugin, add the following to the <path>settings.gradle.kts</path> file:
```

Comment:
Between all the definitions and usages of IJ target products and builds with snapshots and releases, this sentence should emphasize that we are talking here about the plugin version